datastore/prometheus.go

```diff
@@ -146,17 +160,96 @@ func (p *PrometheusExporter) updateMetrics(toPod PodEvent, req Request) {
 	}).Inc()
 }
 
+func (p *PrometheusExporter) handlePackets() {
+	for {
+		select {
+		case <-p.ctx.Done():
+			return
+		case pkt := <-p.pktChanBuffer:
+			p.handlePacket(pkt)
+		}
+	}
+}
+
+func (p *PrometheusExporter) handlePacket(pkt Packet) {
+	labels := prometheus.Labels{
+		"toPort": strconv.Itoa(int(pkt.ToPort)),
+	}
+
+	if pkt.FromType == PodSource {
+		fromPod, found := p.podCache.get(pkt.FromUID)
+		if found {
+			labels["fromPod"] = fromPod.(PodEvent).Name
+			labels["fromAcornApp"] = fromPod.(PodEvent).Labels[appLabel]
+			labels["fromAcornAppNamespace"] = fromPod.(PodEvent).Labels[appNamespaceLabel]
+			labels["fromAcornContainer"] = fromPod.(PodEvent).Labels[containerLabel]
```

Comment:
nit: might save some key-strokes to either:
a) perform the type-assertion for `fromPod` to `PodEvent` once, up-front
b) have `podCache` return a `PodEvent`